package.json

```diff
@@ -60,6 +60,7 @@
   "dependencies": {
     "@capacitor-community/bluetooth-le": "3.0.0",
     "@capacitor/android": "5.0.4",
+    "@capacitor/app": "6.0.0",
```

Comment:
This version of `@capacitor/app` is wrong. Should be closer to 5.0.4 version(other `@capacitor` packages), to not brake anything. And then a major upgrade can be done to Capacitor v6.